src/components/modules/document/common/Upload/Sides/BackSide/BackSideController.js

```diff
@@ -140,6 +131,8 @@ class BackSideController extends VuexModule {
 
       timer.continueProgress(40, 50);
       await documentsService.waitDocumentUpload(docId);
+
+      riskScoringService.sendUserMetrics();
```

Comment:
Add `await` here

Reply:
metrics should be in silent mode and if something goes wrong with service, controller should not throw error